sharedresource/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
```

Comment:
I think approvers should remain as the toplevel api repo approvers.  Reviewers are fine.

Reply:
updated in current push